incskill_site/templates/login.html

```diff
@@ -50,7 +58,35 @@ <h1 id="title">Login</h1>
 <style>
   
 #img{
-  width:150%;
+  width:130%;
+}
+#container {
+	text-align: center;
+}
+
+#text {
+	display: inline-block;
+	
+	color: rgb(255, 255, 255);
+	letter-spacing: 2px;
+}
+
+#cursor {
+	display: inline-block;
+	
+	width: 3px;
+	height: 40px;
```

Comment:
cursor should probably be relative to the current font size so use `em` for this unit

Reply:
hey daniel I cant use const, I have to use var

Reply:
> hey daniel I cant use const, I have to use var

don't use `var`, use `let`; var has a lot of issues with function block scoping and hoisting